CHANGELOG.md

```diff
@@ -114,6 +114,7 @@ gem "capistrano", :github => "capistrano/capistrano"
 
 * Suppress log messages of `git ls-remote` by filtering remote refs (@aeroastro)
 * The Git SCM now allows the repo_url to be changed without manually wiping out the mirror on each target host first (@javanthropus)
+* Default to locking the version using the pessimistic version operator at the patch level.
```

Comment:
This should be under the `master` section, right?

Reply:
Yup. Fixed.